README.md

```diff
@@ -1,43 +1,67 @@
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
+
 # IVR Phone Tree: IVR for beginners. powered by Twilio - Ruby on Rails
 
-This is an application example implementing an automated phone line using Twilio.  For a
-step-by-step tutorial, [visit this link](https://www.twilio.com/docs/howto/walkthrough/ivr-phone-tree/ruby/rails).
+IVRs (interactive voice response) are automated phone systems that can facilitate communication between callers and businesses. If you've ever dialed your credit card company to check on a balance after responding to a series of automated prompts, you've used an IVR. Learn how to build an IVR in minutes using Twilio's powerful TwiML API.
+
+[Read the full tutorial here!](https://www.twilio.com/docs/howto/walkthrough/ivr-phone-tree/ruby/rails)
 
-Deploy this example app to Heroku now!
+## Deploy to Heroku
 
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/TwilioDevEd/ivr-phone-tree-rails)
```

Comment:
Could you please use the `svg` version of the deploy button? IMHO, it looks nicer.